Russian Translation

zumbak04 opened the pull request:
# Changelog:
- Added full Russian translation.

Aehsonneh committed:
Update 000_ru_zuwc_united_localisation.csv

zumbak04 committed:
Province and titles files
(GDC)

Aehsonneh committed:
Update 000_ru_zuwc_united_localisation.csv

Aehsonneh committed:
Update 000_ru_zuwc_united_localisation.csv

zumbak04 committed:
Update 000_ru_zuwc_united_localisation.csv

(GDC)

MoffKadir committed:
Update 000_ru_wc_ruler_titles.csv

MoffKadir committed:
Merge branch 'russian-state-1' of https://github.com/Warcraft-GoA-Development-Team/Warcraft-GoA-Russian into russian-state-1

zumbak04 committed:
New files to translate

(GDC)

MoffKadir committed:
Update 000_ru_wc_ruler_titles_castle.csv

zumbak04 committed:
Merge branch 'russian-state-1' of https://github.com/Warcraft-GoA-Development-Team/Warcraft-GoA-Russian into russian-state-1

MoffKadir committed:
1

MoffKadir committed:
Update 000_ru_wc_ruler_titles_castle.csv

MoffKadir committed:
Update 000_ru_wc_ruler_titles_city.csv

MoffKadir committed:
Update 000_ru_wc_ruler_titles_nomadic.csv

Aehsonneh committed:
Update 000_ru_wc_cultures.csv

Aehsonneh committed:
Merge branch 'russian-state-1' of https://github.com/Warcraft-GoA-Development-Team/Warcraft-GoA-Russian into russian-state-1

MoffKadir committed:
Update 000_ru_wc_ruler_titles_technocracy.csv

MoffKadir committed:
Update 000_ru_wc_ruler_titles_temple.csv

MoffKadir committed:
Update 000_ru_wc_ruler_titles_tribal.csv

MoffKadir committed:
localisation

MoffKadir committed:
Update 000_ru_wc_ruler_titles_nomadic.csv

MoffKadir committed:
Update 000_ru_Province_names.csv

MoffKadir committed:
Update 000_ru_Province_names.csv

MoffKadir committed:
Update 000_ru_wc_ruler_titles_nomadic.csv

MoffKadir committed:
Update 000_ru_Province_names.csv

MoffKadir committed:
Update 000_ru_Province_names.csv

MoffKadir committed:
Update 000_ru_Province_names.csv

MoffKadir committed:
Update 000_ru_Province_names.csv

Aehsonneh committed:
Update 000_ru_wc_cultures.csv

MoffKadir committed:
Update 000_ru_wc_artifact_localisation.csv

MoffKadir committed:
Update 000_ru_wc_artifact_localisation.csv

MoffKadir committed:
Update 000_ru_wc_artifact_localisation.csv

MoffKadir committed:
Update 000_ru_wc_artifact_localisation.csv

MoffKadir committed:
Update 000_wc_culture_buildings.csv

MoffKadir committed:
Update 000_wc_culture_buildings.csv

MoffKadir committed:
Update 000_wc_culture_buildings.csv

MoffKadir committed:
Update 000_wc_culture_buildings.csv

MoffKadir committed:
Update 000_wc_culture_buildings.csv

MoffKadir committed:
Update 000_wc_culture_buildings.csv

MoffKadir committed:
Update 000_wc_culture_buildings.csv

MoffKadir committed:
Update 000_ru_Province_names.csv

MoffKadir committed:
Update 000_ru_Province_names.csv

MoffKadir committed:
Update 000_ru_Province_names.csv

MoffKadir committed:
Update 000_ru_Province_names.csv

MoffKadir committed:
Update 000_ru_Province_names.csv

MoffKadir committed:
Update 000_ru_Province_names.csv

MoffKadir committed:
Update 000_ru_Province_names.csv

MoffKadir committed:
Update 000_ru_Province_names.csv

MoffKadir committed:
Update 000_wc_ru_wonders_upgrades.csv

MoffKadir committed:
localisation

MoffKadir committed:
Update 000_ru_wc_traits.csv

MoffKadir committed:
Update 000_ru_wc_traits.csv

MoffKadir committed:
Update 000_ru_wc_religions.csv

MoffKadir committed:
Update 000_ru_zuwc_united_localisation.csv

MoffKadir committed:
Update 000_ru_zuwc_united_localisation.csv

MoffKadir committed:
Update 000_ru_Province_names.csv

MoffKadir committed:
Update 000_ru_Province_names.csv

MoffKadir committed:
Update 000_ru_Province_names.csv

MoffKadir committed:
Update 000_ru_Province_names.csv

MoffKadir committed:
Update 000_ru_Province_names.csv

MoffKadir committed:
Update 000_ru_Province_names.csv

MoffKadir committed:
Update 000_ru_Province_names.csv

MoffKadir committed:
Update 000_ru_Province_names.csv

zumbak04 committed:
Small renames

(GDC)

zumbak04 committed:
New files

(GDC)

MoffKadir committed:
Update 000_ru_wc_vanilla_replacements.csv

MoffKadir committed:
Update 000_ru_wc_vanilla_replacements.csv

MoffKadir committed:
Update 000_ru_wc_job_titles.csv

MoffKadir committed:
Update 000_ru_ciwc_united_localisation.csv

MoffKadir committed:
Update 000_ru_wc_military_localization.csv

MoffKadir committed:
Update 000_ru_wc_military_localization.csv

MoffKadir committed:
Update 000_ru_memewc_united_localisation.csv

zumbak04 committed:
Moved province names

(GDC)

MoffKadir committed:
Update 000_ru_wc_vanilla_replacements.csv

zumbak04 committed:
Create 0000_ru_Other_title_names.csv

(GDC)

MoffKadir committed:
Update 0000_ru_Other_title_names.csv

MoffKadir committed:
Update 000_ru_wc_ruler_titles.csv

zumbak04 committed:
Renamed a few files

(GDC)

zumbak04 committed:
New files

(GDC)

MoffKadir committed:
small fixes

MoffKadir committed:
russian localisation of titles

MoffKadir committed:
Update 000_ru_Duchy_names.csv

MoffKadir committed:
Update 000_ru_wc_economy.csv

MoffKadir committed:
Update 0_ru_tyr_wc_loa.csv

Aehsonneh committed:
Update 000_ru_wc_sad.csv

MoffKadir committed:
Update 0_ru_tyr_wc_loa.csv

MoffKadir committed:
Update 0_ru_wc_uniques.csv

MoffKadir committed:
Update 000_ru_wc_offmap.csv

MoffKadir committed:
Update 0_ru_wc_uniques.csv

MoffKadir committed:
Update 000_ru_zuwc_united_localisation.csv

MoffKadir committed:
loc fix

MoffKadir committed:
fix

MoffKadir committed:
Update 0_ru_wc_religion_buildings.csv

MoffKadir committed:
Update 000_ru_wc_mnm_replace.csv

MoffKadir committed:
loc

MoffKadir committed:
Update 000_ru_wc_generics.csv

MoffKadir committed:
artifact loc

MoffKadir committed:
Update 000_ru_wc_random_artifact_names.csv

zumbak04 committed:
Update 000_ru_Province_names.csv

Purged other languages.
(GDC)

zumbak04 committed:
Update 000_ru_wc_ruler_titles_castle.csv

Purged other languages.
(GDC)

zumbak04 committed:
Purged other languages

(GDC)

zumbak04 commented on localisation/000_ru_Empire_names.csv:
```diff
@@ -0,0 +1,41 @@
+#CODE;RUSSIAN;;;;;;;;;;;;;x
```
GetAdjective используются в зданиях и должны быть прилагательными.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/81562991-bc3a1a00-93a6-11ea-8a56-4c0be5a1f23f.png)
![image](https://user-images.githubusercontent.com/46384992/81562999-bfcda100-93a6-11ea-9ad5-6474b30903fc.png)

</details>

zumbak04 replied:
Названия культур... странные.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/81563056-d116ad80-93a6-11ea-86ae-ef79eb35bd8e.png)

</details>

zumbak04 commented on localisation/000_ru_Empire_names.csv:
```diff
@@ -0,0 +1,41 @@
+#CODE;RUSSIAN;;;;;;;;;;;;;x
```
Где написано Cap нужно писать с заглавной.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/81563222-07542d00-93a7-11ea-96e6-f96099b46277.png)
![image](https://user-images.githubusercontent.com/46384992/81563579-7c276700-93a7-11ea-8874-dfedd4ee6ff9.png)

</details>

MoffKadir replied:
Исправлено 02.06.2020

zumbak04 committed:
Small fixes

(GDC)

MoffKadir committed:
(RUS) added and translated missing lines

MoffKadir committed:
Update 000_ru_zuwc_united_localisation.csv

MoffKadir committed:
Update 000_ru_wc_artifact_localisation.csv

MoffKadir committed:
added new lines

MoffKadir committed:
-

MoffKadir committed:
loc

MoffKadir committed:
rus

MoffKadir committed:
Update 000_ru_wc_vanilla_replacements.csv

MoffKadir committed:
fix

MoffKadir committed:
Update 000_ru_zuwc_united_localisation.csv

MoffKadir committed:
loc

MoffKadir committed:
adapt

MoffKadir committed:
fix

MoffKadir committed:
russian localisation of Gorian Empire feature and other fixes

zumbak04 committed:
Updated religion names
Updated some religion names.

zumbak04 committed:
A few more religions

Updated a few more religion names.

MoffKadir committed:
Update 0000_ru_Other_title_names.csv

zumbak04 committed:
Suggestions and proofreading

(GDC)

Co-Authored-By: Mikekz <mikekz@users.noreply.github.com>

zumbak04 committed:
A few more suggestions

(GDC)

Co-Authored-By: Mikekz <mikekz@users.noreply.github.com>

MoffKadir committed:
loc

MoffKadir committed:
l

zumbak04 committed:
Revert "l"

This reverts commit 2044d3ccf37b391135a1f6385ed0cd68797fc509, reversing
changes made to 6174cd5469253dacc59aa1eb3c46941e222302df.
(GDC)

MoffKadir committed:
Revert "l"

This reverts commit 2044d3ccf37b391135a1f6385ed0cd68797fc509, reversing
changes made to 84827cabddedac030d6b9250d1dd1e1dd4d780e9.

MoffKadir committed:
Revert "Suggestions and proofreading"

This reverts commit 5e1d94a31f6c7b6b0390463a7588b55d2664f0aa.

MoffKadir committed:
Revert "Revert "Suggestions and proofreading""

This reverts commit 61091c1304ea51b6d07aa71420c8a9d497c40d4a.

MoffKadir committed:
restoration of files

zumbak04 committed:
Cult of Loken => Loken

(GDC)

MoffKadir committed:
small fixes

Локена - Локенизм
Артес - Артас

MoffKadir committed:
Update 000_ru_zuwc_united_localisation.csv

MoffKadir committed:
Локализация

Аспекты Элуны

MoffKadir committed:
upd

MoffKadir committed:
upd

MoffKadir committed:
upd

MoffKadir committed:
upd

MoffKadir committed:
Update 00_ru_wc_sacrifice_events.csv

MoffKadir committed:
Upd

адаптация герметистов

MoffKadir committed:
Update 000_ru_wc_vanilla_replacements.csv

MoffKadir committed:
upd

MoffKadir committed:
Тест

Тест локализации сгенерированного оружия

MoffKadir committed:
upd

MoffKadir committed:
upd

MoffKadir committed:
upd

tcbydev committed:
Update 000_ru_ciwc_united_localisation.csv

tcbydev committed:
Update 000_ru_wc_traits.csv

tcbydev committed:
Update 000_ru_wc_sad.csv

tcbydev committed:
Update 0000_ru_Other_title_names.csv

Болотноплавниковые -> Болотный Плавник
Страж Тирисфаля -> Хранитель Тирисфаля
Песня Войны -> Песни Войны
Смеющийся Череп -> Веселый Череп
Кровавая Лапа -> Кроволапы
Темное Копье -> Черное Копье
Флибустьеры Северного моря -> Флибустьеры Северных Морей

tcbydev committed:
Update 000_ru_Duchy_names.csv

Остров Новолунья -> Остров Новолуния
Острова Темного Копья -> Острова Черного Копья

tcbydev committed:
Update 000_ru_Empire_names.csv

Империя Аратора -> Аратор

tcbydev committed:
Update 000_ru_wc_artifact_localisation.csv

Сталебой -> Железогубитель
Волна огня -> Огненный столб

tcbydev committed:
Update 000_ru_wc_ruler_titles.csv

Мастер -> Мажордом
Повелитель -> Мастер
Флотмейстер -> Командир флота
Военный магистр -> Воевода
Великий магистр -> Верховный магистр
Пророк крови -> Кровавый пророк
Пророчица крови -> Кровавая пророчица
Верховный терномант -> Верховный терноплет
Темный архиепископ -> Сумеречный архиепископ
Повелитель -> Властитель
Повелительница -> Властительница

tcbydev committed:
Update 000_ru_wc_ruler_titles_castle.csv

Верховный повелитель -> Верховный владыка
Верховная повелительница -> Верховная владычица

tcbydev committed:
Update 0000_ru_Other_title_names.csv

Исправление кодировки

tcbydev committed:
Upd

tcbydev committed:
upd

tcbydev committed:
upd

tcbydev committed:
Update 0_ru_tyr_wc_loa.csv

tcbydev committed:
Update 0_ru_wc_uniques.csv

tcbydev committed:
Update 000_ru_zuwc_united_localisation.csv

tcbydev committed:
upd

tcbydev committed:
Update 000_ru_wc_vanilla_replacements.csv

tcbydev committed:
Update 000_ru_zuwc_united_localisation.csv

zumbak04 committed:
ANSI encoding

Encoding must be ANSI.
(GDC)

zumbak04 committed:
Reverted a few lines

Mostly related to province names.
(GDC)

zumbak04 committed:
Restored a few deleted duplicates

(GDC)

tcbydev committed:
Update 000_ru_zuwc_united_localisation.csv

tcbydev committed:
Update 0_ru_wc_cultures.csv

tcbydev committed:
upd

tcbydev committed:
Update 000_ru_County_names.csv

tcbydev committed:
test

tcbydev committed:
Revert "Update 0_ru_wc_cultures.csv"

This reverts commit 0dc74cec8363c724531f3a66e616d905618ee6f0.

tcbydev committed:
Update 000_ru_zuwc_united_localisation.csv

tcbydev committed:
Update 000_ru_zuwc_united_localisation.csv

tcbydev committed:
Update 000_ru_zuwc_united_localisation.csv

tcbydev committed:
Update 000_ru_zuwc_united_localisation.csv

tcbydev committed:
loc

tcbydev committed:
upd

tcbydev committed:
Update 000_ru_County_names.csv

tcbydev committed:
upd

tcbydev committed:
upd

tcbydev committed:
upd

tcbydev committed:
fix

tcbydev committed:
upd

tcbydev committed:
upd

tcbydev committed:
upd

tcbydev committed:
upd

tcbydev committed:
upd

tcbydev committed:
upd

tcbydev committed:
upd

tcbydev committed:
Update 0_ru_wc_culture_buildings.csv

tcbydev committed:
upd

tcbydev committed:
upd

tcbydev committed:
upd

tcbydev committed:
upd

MoffKadir committed:
Upd.

Добавлены новые строчки

tcbydev committed:
upd

tcbydev committed:
Update 000_ru_County_names.csv

MoffKadir committed:
Update 000_ru_zuwc_united_localisation.csv

tcbydev committed:
fix

tcbydev committed:
Update 000_ru_zuwc_united_localisation.csv

tcbydev committed:
Update 000_ru_Barony_names.csv

tcbydev committed:
Update 000_ru_Barony_names.csv

tcbydev committed:
Update 000_ru_Barony_names.csv

tcbydev committed:
Update 000_ru_Barony_names.csv

tcbydev committed:
Update 000_ru_County_names.csv

tcbydev committed:
del

zumbak04 committed:
Merge remote-tracking branch 'remotes/origin/russian-state-1' into pr/5

# Conflicts:
#	localisation/000_ru_wc_artifact_localisation.csv
#	localisation/000_ru_wc_sad.csv
#	localisation/000_ru_zuwc_united_localisation.csv

zumbak04 committed:
Small fixes
(GDC)

zumbak04 committed:
Merge pull request #5 from tcbydev/russian-state-1

Russian state 1

MoffKadir committed:
Update 000_ru_zuwc_united_localisation.csv

MoffKadir committed:
Upd.

MoffKadir committed:
Update 000_ru_wc_vanilla_replacements.csv

MoffKadir committed:
upd.

MoffKadir committed:
Update 000_ru_wc_mod_version.csv

MoffKadir committed:
Update 000_ru_zuwc_united_localisation.csv
